.github/workflows/build.yml

```diff
@@ -1,28 +1,80 @@
 name: 🏗️ Data Build
 
 on:
-  issues:
-    types: [opened]
+  workflow_dispatch:
+    inputs:
+      build_note:
+        description: "Note about this build"
+        type: string
+        required: false
+      logging_level:
+        description: "Logging level"
+        type: choice
+        required: true
+        options:
+          - WARNING
+          - INFO
+          - DEBUG
+  workflow_call:
+    inputs:
+      logging_level:
+        required: true
+        type: string
+env:
+  LOGGING_LEVEL: ${{ inputs.logging_level }}
+  BUILD_ENGINE: postgresql://postgres:postgres@localhost:5432/postgres
+  DO_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+  DO_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+  DO_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
 
 jobs:
   build:
-    if: >-
-      contains(github.event.issue.labels.*.name, 'data build')
+    name: Build Dataset
     runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgis/postgis:12-3.0-alpine
```

Comment:
Another update we've neglected but might be worth exploring if we can upgrade this postgis image as well